pages/linux/pdfxup.md

```diff
@@ -0,0 +1,16 @@
+# pdfxup
+
+> N-up PDF pages.
```

Comment:
What is N-up?

Reply:
Putting several pages on a single page by scaling / rotation
https://en.wikipedia.org/wiki/N-up

Might be vocabulary known for people who have looked up the tool. Or we could add a short description.

Reply:
For those who don't know the term, it would be helpful to explain it.
```suggestion
> N-up PDF pages.
> N-upping means putting multiple pages onto one page by scaling and rotating them.
```